src/Sulu/Bundle/PageBundle/Tests/Functional/Controller/PageControllerTest.php

```diff
@@ -69,6 +69,7 @@ class PageControllerTest extends SuluTestCase
     public function setUp(): void
     {
         $this->client = $this->createAuthenticatedClient();
+        $this->client->disableReboot(); // see https://github.com/symfony/symfony/issues/45580
```

Comment:
this change is unrelated to the change in this PR. as the test already fails on 2.4 branch: https://github.com/sulu/sulu/commit/9561f75d81d89c8ced7733578525977457e63d2f